packages/quantic/force-app/main/default/lwc/quanticResultLink/quanticResultLink.js

```diff
@@ -1,20 +1,23 @@
-import {LightningElement, api} from 'lwc';
-import {
-  getHeadlessBundle,
-  getHeadlessEnginePromise,
-} from 'c/quanticHeadlessLoader';
-import {ResultUtils} from 'c/quanticUtils';
+import { getHeadlessBundle, getHeadlessEnginePromise } from 'c/quanticHeadlessLoader';
+import { ResultUtils } from 'c/quanticUtils';
+import { NavigationMixin } from 'lightning/navigation';
+import { LightningElement, api } from 'lwc';
+
+import opensInBrowserTab from '@salesforce/label/c.quantic_OpensInBrowserTab';
+import opensInSalesforceSubTab from '@salesforce/label/c.quantic_OpensInSalesforceSubTab';
 
 /** @typedef {import("coveo").Result} Result */
 /** @typedef {import("coveo").SearchEngine} SearchEngine */
 
 /**
  * The `QuanticResultLink` component creates a clickable link from a result that points to the original item.
+ * If the result is of a Salesforce record or a Salesforce Knowledge article it will open the link in a new salesforce console subtab.
```

Comment:
```suggestion
 * If the result is a Salesforce record or a Salesforce Knowledge article it will open the link in a new salesforce console subtab.
```